ignition-server/cmd/start.go

```diff
@@ -149,6 +149,10 @@ func setUpPayloadStoreReconciler(ctx context.Context, registryOverrides map[stri
 		return nil, fmt.Errorf("unable to create image file cache: %w", err)
 	}
 
+	imageMetaDataProvider := &util.RegistryClientImageMetadataProvider{
```

Comment:
why does the ignition-server needs the new metadata provider? Isn't this which is already available all we need https://github.com/openshift/hypershift/blob/main/ignition-server/cmd/start.go#L155-L165? so the you can invoke GetOpenShiftImageRegistryOverrides?

Reply:
The releaseProvider does not contain the ImageMetadata of the Release image itself, but the content. And to verify the Arch functions will need all the metadata including the layers.